### DISCUSSION TITLE
pick another browser? on load up

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2547] [Q&A] [answered]
so i have a imac 27 inch that i use for all my daily needs and a small 24 inch to my right that i use just for fooocus
when i run the file , of course it opens a new tab in firefox and then i drag that tab to the right screen

can i somehow tell it to use lets say chrome on start up? as i can make any app loads on desktop 1 or 2 from my deck.
seen a few site saying about opening chrome, but of course needs to be 127 port

**Comment by mashb1t** [comment, marked as answer]
No, this is currently not possible. Fooocus will use your default browser on startup, but feel free to open any other browser, copy the URL over and use it there.

**Comment by XDM-Inc** [comment]
Another good workaround is to change your launch option to --disable-in-browser and then (i used a portable chrome for this)
go to chrome and paste the url and then when its loaded go to the 3 dot menu and open "cast,save and share" menu and "install" or "make a shortcut" for that site. this make a near perfect app like feeling for fooocus without url bar, bookmarks in the way.
for linux you can make a .desktop file that launches chrome and add this to the arguments
`--app=http://127.0.0.1:7865/`  This command might work from windows as well if you use a bat file or command
`path/to/chrome.exe --app=http://127.0.0.1:7865`
